Soccar [Client]/src/nl/soccar/library/Entity.java

```diff
@@ -0,0 +1,35 @@
+package nl.soccar.library;
+
+public abstract class Entity {
+    
+    protected double x;
+    protected double y;
+    protected double degree;
+    protected double velocity;
+
+    public Entity(double x, double y, double degree) {
+        this.x = x;
+        this.y = y;
+        this.degree = degree;
+        velocity = 0;
+    }
+    
+    public abstract void move(double x, double y, double degree);

```

Comment:
Even kijken of we dit ook daadwerkelijk zo willen implementeren, op zich is dit voor nu prima.